### PR TITLE
GUACAMOLE-101: Documentation for LDAP User Filtering

### PR DESCRIPTION
**Pull request by necouchman**
Added documentation for the two properties added for supporting filtering of users and connections in LDAP.

### CODE REVIEW

**Comment by necouchman** [comment]
Removed documentation related to connection filtering, as that has been removed from the code.

File: src/chapters/ldap-auth.xml [hunk 1]
```diff
@@ -384,6 +384,15 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                                 have no effect on Guacamole configurations.</emphasis></para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-user-search-filter</property></term>
+                    <listitem>
+                        <para>The search filter used to query the LDAP tree for users that
+                            can log into and be granted privileges in Guacamole.  <emphasis>If
+                            this property is omitted the default of (objectClass=*) will be used.
```
Comment (mike-jumper) [inline src/chapters/ldap-auth.xml]: Please place `(objectClass=*)` within quotes, so it's clear that it's a literal value.

Comment (necouchman) [inline src/chapters/ldap-auth.xml]: Fixed.

**Comment by mike-jumper** [comment]
Please rebase - things conflict against current `master`.

**Comment by necouchman** [comment]
Rebased.

File: src/chapters/ldap-auth.xml [hunk 2]
```diff
@@ -395,6 +395,15 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                             aliases.</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-user-search-filter</property></term>
```
Comment (mike-jumper) [inline src/chapters/ldap-auth.xml]: OK - I built the manual from these changes and noticed that the organization of this section is slightly off. There's a section in the middle of this area which is devoted to users and search:

> ldap-user-search-stuff

but the new `ldap-user-search-filter` property documentation is simply appended to the end of the section among other unrelated properties:

> ldap-user-search-filter-section

Perhaps this should be moved up a bit?

Comment (necouchman) [inline src/chapters/ldap-auth.xml]: Yeah, that makes sense.  I moved it just below the ldap-username-attribute.

**Pull request merged**